GitUI/UserControls/FileStatusList.cs

```diff
@@ -1467,9 +1435,9 @@ private void FileStatusListView_MouseMove(object sender, MouseEventArgs e)
                 {
                     var fileList = new StringCollection();
 
-                    foreach (GitItemStatus item in SelectedItems)
+                    foreach (var item in SelectedItems)
```

Comment:
This change is undesirable as it reduces comprehensibility of the code.

Reply:
I thought the recommendation was to use ```var``` where type was defined, but I can change (to FileStatusItem in these examples). Not sure if there are more though.